docs/demo/demo.rst

```diff
@@ -428,15 +428,30 @@ Admonitions
 
    You can make up your own admonition too.
 
-.. admonition:: If you add a name flag, it will be styled
-   :name: warning
+.. admonition:: If you add a class flag, it will be styled
+   :class: warning
```

Comment:
this is nice - we don't need to use our hacky extra code to achieve this effect now - this is functionality that "already works" in Sphinx